frontend/src/components/SemiCircleBg.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="stack-container">
+    <div class="semi-circle" v-bind:style="color" />
```

Comment:
i think the RHS of `v-bind:style="color" ` must be some kind of pseudo css. e.g. `{ backgroundColor: color }`